modules/mirror-registry-flags.adoc

```diff
@@ -13,11 +13,11 @@ The following flags are available for the _mirror registry for Red Hat OpenShift
 | `--autoApprove` | A boolean value that disables interactive prompts. If set to `true`, the `quayRoot` directory is automatically deleted when uninstalling the mirror registry. Defaults to `false` if left unspecified.
 | `--initPassword` | The password of the init user created during Quay installation. Must be at least eight characters and contain no whitespace.
 |`--initUser string` | Shows the username of the initial user. Defaults to `init` if left unspecified.
-| `--quayHostname` | The fully-qualified domain name of the mirror registry that clients will use to contact the registry. Equivalent to `SERVER_HOSTNAME` in the Quay `config.yaml`. Must resolve by DNS. Defaults to `<targetHostname>:8443` if left unspecified. ^[1]^
+| `--quayHostname` | The fully-qualified domain name of the mirror registry that clients will use to contact the registry. Equivalent to `SERVER_HOSTNAME` in the Quay `config.yaml`. Must resolve by DNS. Defaults to `<targetHostname>:8443` if left unspecified. ^[1]^, ^[2]^
```

Comment:
Not critical, but just wondering if there's a reason that this had to be added as a second note, instead of just adding the sentence to the existing note # 1?

Reply:
You're right. At first I thought they were slightly unrelated, but now I think they're kind of related.